cmd/ww/server.go

```diff
@@ -320,6 +320,10 @@ func server(args ...string) {
 		// https://bugs.webkit.org/show_bug.cgi?id=201591
 		w.Header().Set("Content-Security-Policy", "default-src 'self'; script-src 'self' 'unsafe-eval'; img-src 'self' blob:; connect-src 'self' ws://localhost/ wss://tip.webwormhole.io/ wss://webwormhole.io/")
 
+		// Set HSTS header
+		// https://ssl-config.mozilla.org/#server=go&config=intermediate
+		w.Header().Set("Strict-Transport-Security", "max-age=63072000")
```

Comment:
this might (or at least used to?) be problematic when running on localhost with no https for dev. can we wrap it in a check for r.URL.Scheme == "https"?

couple of nits:
- update the comment to `Set HSTS header for 2 years.`
- i think the mozilla urls make more sense to keep in the commit message.

Reply:
Ok .. I added https check you proposed.

Reply:
I just checked and it turns out r.URL.Scheme isn't actually populated by the http.Server, so we can't use that. We can use httpsaddr instead.

Sorry about the detour. I'll merge this anyway and change it to httpsaddr in a followup commit.